rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -996,6 +996,54 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
+    protected void processValidationFailure(Federation proposedFederation) {
+        eventLogger.logCommitFederationFailure(rskExecutionBlock, proposedFederation);
+        logger.warn("[processValidationFailure] Proposed federation validation failed so svp values will be cleared.");
+        clearSvpValues();
+    }
+
+    private void clearSvpValues() {
+        federationSupport.clearProposedFederation();
+
+        String methodName = "[clearSvpValues]";
```

Comment:
I understand the idea, but IMO this seems not necessary it should be fine to just have it as part of the log statements directly. But feel free to leave it as is.

Reply:
yes i think youre right. It looks cleaner the way you say. Thanks :)

Reply:
updated